src/CMakeLists.txt

```diff
@@ -32,23 +41,34 @@ execute_process(
 
 message(STATUS "Awkward includes : ${AWKWARD_INCLUDE}")
 message(STATUS "Awkward libraries: ${AWKWARD_LIBRARIES}")
-include_directories(BEFORE "${AWKWARD_INCLUDE}")
```

Comment:
No global commands, please!

Reply:
This comes from here: https://github.com/scikit-hep/awkward-1.0/blob/81e6848405dca729ee2929f06fa17e17d05eca54/dependent-project/CMakeLists.txt#L15

So it probably should be fixed there as well? I was wondering what this was doing.

Reply:
Awkward needs work on this, I've been planning to rework this for a while now (as @jpivarski knows). It should also provide nice targets so that it's really easy to use. I've been trying to get a few things inplace for tackling that (would like to add a utility to wheel but the person I want to review it is on a 2 week vacation).

Yes, there should never be directory level commands like this. They are not transitive to any other places that the target might be used, harder to track down, and promote bad (non-target) based thinking.

Reply:
Awkward should ship it's config files in the wheel, as well, like pybind11. I'm hoping to improve that too with the cmake and scikit-build packages, but probably not till next year at the earliest.